pkg/controller/plan/tls_utils.go

```diff
@@ -50,23 +51,30 @@ func ComposeTLSSecret(namespace, clusterName, componentName string) (*v1.Secret,
 		SetStringData(map[string]string{}).
 		GetObject()
 
-	const tpl = `{{- $cert := genCA "KubeBlocks" 36500 }}
-{{ $cert.Cert }}
-{{ $cert.Key }}
-`
-	out, err := buildFromTemplate(tpl, nil)
+	// use ca gen cert
+	// IP: 127.0.0.1
+	// DNS: localhost and *.<clusterName>-<componentName>-headless.<namespace>.svc.cluster.local
+	const spliter = "___spliter___"
+	SignedCertTpl := fmt.Sprintf(`
+	{{- $ca := genCA "KubeBlocks" 36500 -}}
+	{{- $cert := genSignedCert "%s peer" (list "127.0.0.1") (list "localhost" "*.%s-%s-headless.%s.svc.cluster.local") 36500 $ca -}}
```

Comment:
add "::1" to ip list to support ipv6,  like (list "127.0.0.1" "::1")

Reply:
solved